apps/wallet-ui/src/app/[ecosystem]/login/page.tsx

```diff
@@ -0,0 +1,14 @@
+"use client";
+import { client } from "@/lib/client";
+import { ConnectEmbed, useActiveAccount } from "thirdweb/react";
+import { inAppWallet } from "thirdweb/wallets";
+
+export default function Page() {
+  return (
+    <ConnectEmbed
+      autoConnect={true}
+      wallets={[inAppWallet()]}
```

Comment:
would have to be ecosystem wallet right? with the correct ID etc?

Reply:
Yep this is temporary